GLAA.Services/Admin/AdminUserPostDataHandler.cs

```diff
@@ -32,6 +39,20 @@ public string Insert(AdminUserViewModel model)
 
             userManager.AddToRoleAsync(user, model.Role).GetAwaiter().GetResult();
 
+            // For more information on how to enable account confirmation and password reset please
+            // visit https://go.microsoft.com/fwlink/?LinkID=532713
+            var code = userManager.GeneratePasswordResetTokenAsync(user).GetAwaiter().GetResult();
```

Comment:
We definitely need to look at this in more detail, at some point, 99.9% sure the async stuff _should_ work.

Reply:
Yeah, it just isn't working. As I understand it should automatically wait for an `await`ed operation to complete when the result is needed but it doesn't.